src/style.css

```diff
@@ -1 +1,57 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
```

Comment:
The font-weight '300' is used here, which is typically considered a light weight. Ensure that this weight is supported by the font file 'Avenir-Book.ttf'. If not, consider using a more standard weight like 'normal' or 'bold'.